Code/Sample/Common/main_glfw_gl3.cpp

```diff
@@ -0,0 +1,335 @@
+// Dear ImGui: standalone example application for GLFW + OpenGL 3, using programmable pipeline
```

Comment:
All files in the hierarchy are added to the build. This file doesn't compile under Windows without OpenGL lib setup `Cannot open include file: 'GLFW/glfw3.h'`. The cpp file content needs to be wrapped in a define so that its content is only active with a OPENGL define (and probably something similar for the default DirectX samples).

Reply:
Ok,  I added a #ifdef on top of main_glfw_gl3.cpp and main.cpp